### PR TITLE
Fix verify-commits on OSX, update for new bad Tree-SHA512, point travis to different keyservers

### PR DESCRIPTION
**Pull request by TheBlueMatt**
This should (again) fix all known issues in verify-commits

### CODE REVIEW

File: contrib/verify-commits/verify-commits.sh [hunk 1]
```diff
@@ -37,92 +56,77 @@ IS_SIGNED () {
...
 	fi
 
-	if [ "${REVSIG_ALLOWED#*$1}" != "$REVSIG_ALLOWED" ]; then
+	if [ "${REVSIG_ALLOWED#*$CURRENT_COMMIT}" != "$REVSIG_ALLOWED" ]; then
 		export BITCOIN_VERIFY_COMMITS_ALLOW_REVSIG=1
 	else
 		export BITCOIN_VERIFY_COMMITS_ALLOW_REVSIG=0
 	fi
 
 	if ! git -c "gpg.program=${DIR}/gpg.sh" verify-commit $1 > /dev/null; then
```
Comment (jonasschnelli) [inline contrib/verify-commits/verify-commits.sh]: I think $1 shouldn't be here. I think you have to change $1 to $CURRENT_COMMIT?

**Comment by jonasschnelli** [comment]
Worked for me after I changed `$1` to `$CURRENT_COMMIT` on L66.

File: contrib/verify-commits/verify-commits.sh (hunk 1 above)
Comment (TheBlueMatt) [inline contrib/verify-commits/verify-commits.sh]: Grr, you're right, fixed.

**Comment by jonasschnelli** [comment]
Tested ACK df5bae2e7b0ccd2b983f19123b0d4b619ea2e598

**Pull request merged**